File: Booking controller.java
import org.springframework.web.bind.annotation.*;

import java.util.List;

@RestController
public class BookingController {
    @PostMapping("/book")
    public String book(@RequestBody String booking) {
        return "{\"status\":\"success\",\"message\":\"Booking confirmed!\"}";
    }

    @PostMapping("/pay")
    public String pay(@RequestBody String booking) {
        return "{\"status\":\"success\",\"message\":\"Payment updated!\"}";
    }

    @GetMapping("/bookings")
    public List<String> bookings() {
        return List.of();
    }
}

File: Booking controller.kt
import org.springframework.web.bind.annotation.*

@RestController
class BookingController {
    @PostMapping("/book")
    fun book(@RequestBody booking: String) =
        mapOf("status" to "success", "message" to "Booking confirmed!")

    @PostMapping("/pay")
    fun pay(@RequestBody booking: String) =
        mapOf("status" to "success", "message" to "Payment updated!")

    @GetMapping("/bookings")
    fun bookings() = listOf<String>()
}